libs/src/services/creatorNode/CreatorNode.ts

```diff
@@ -469,86 +587,101 @@ class CreatorNode {
           immediate
         }
       }
-      return axios(req)
+      return await axios(req)
     }
+    return
   }
 
   /* ------- INTERNAL FUNCTIONS ------- */
 
   /**
    * Signs up a creator node user with a wallet address
-   * @param {string} walletAddress
+   * @param walletAddress
    */
-  async _signupNodeUser (walletAddress) {
-    await this._makeRequest({
-      url: '/users',
-      method: 'post',
-      data: { walletAddress }
-    }, false)
+  async _signupNodeUser(walletAddress: string) {
+    await this._makeRequest(
+      {
+        url: '/users',
+        method: 'post',
+        data: { walletAddress }
+      },
+      false
+    )
   }
 
   /**
    * Logs user into cnode, if not already logged in.
    * Requests a challenge from cnode, sends signed challenge response to cn.
    * If successful, receive and set authToken locally.
    */
-  async _loginNodeUser () {
+  async _loginNodeUser() {
     if (this.authToken) {
       return
     }
 
     const walletPublicKey = this.web3Manager.getWalletAddress()
     let clientChallengeKey
-    let url
+    let url: string | undefined
 
     try {
-      const challengeResp = await this._makeRequest({
-        url: '/users/login/challenge',
-        method: 'get',
-        params: {
-          walletPublicKey
-        }
-      }, false)
+      const challengeResp = await this._makeRequest(
+        {
+          url: '/users/login/challenge',
+          method: 'get',
+          params: {
+            walletPublicKey
+          }
+        },
+        false
+      )
 
       clientChallengeKey = challengeResp.data.challenge
       url = '/users/login/challenge'
     } catch (e) {
       const requestUrl = this.creatorNodeEndpoint + '/users/login/challenge'
-      await this._handleErrorHelper(e, requestUrl)
+      await this._handleErrorHelper(e as Error, requestUrl)
     }
 
     const signature = await this.web3Manager.sign(clientChallengeKey)
```

Comment:
can we just move this block into the try {}?

Reply:
yeah i think that's a good call cause the catch will just rethrow anyway yeah?